.gitignore

```diff
@@ -2,4 +2,10 @@ lib
 obj
 bin
 .DS_Store
-.#*#
\ No newline at end of file
+<<<<<<< HEAD
+.#*#
+a_dot
+moving_triangle
+=======
+.#*#
+>>>>>>> flyx/master
```

Comment:
I don't really understand the Git message in this file.
I've hopefully fixed it by copying the contents of the flyx/master file.
What is the .#*# for?